### PR TITLE
Dr. Elephant init action

### PR DESCRIPTION
**Pull request by adam-stasiak**
This init action is used for Dr. Elephant initialization.

In order to use this action it is needed to use dr-elephant-env.sh first.

```test.sh``` script can be used for installation Dr. Elephant on most popular cluster configurations and submitting sample spark jobs on them.

### CODE REVIEW

File: dr-elephant/dr-elephant-env.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# This script is used for environment setup for dr-elephant
```
Comment (karth295) [inline dr-elephant/dr-elephant-env.sh]: Can this just be combined with the main script?

File: dr-elephant/dr-elephant-env.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+set -x -e
+# Install on master node
+[[ "$(/usr/share/google/get_metadata_value attributes/dataproc-role)" == 'Master' ]] || exit 0
+touch ~/.bashrc
```
Comment (karth295) [inline dr-elephant/dr-elephant-env.sh]: Nit: `touch` is unnecessary -- `cat` will create the file if necessary

File: dr-elephant/dr-elephant-env.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+cat << 'EOF' > ~/.bashrc
```
Comment (karth295) [inline dr-elephant/dr-elephant-env.sh]: Nit: append to `~/.bashrc`, rather than overriding.

File: dr-elephant/dr-elephant.sh [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+  # Build dr elephant and move outputs
+  bash ./compile.sh compile.conf
+  unzip /root/dr-elephant/dist/dr-elephant-2.1.7.zip -d /root/dr-elephant/dist/
+}
+
+function install_mysql(){
+  # Install MySQL
+
+  export DEBIAN_FRONTEND=noninteractive
+  curl -LO https://dev.mysql.com/get/mysql-apt-config_0.7.3-1_all.deb
```
Comment (karth295) [inline dr-elephant/dr-elephant.sh]: Isn't mysql or mariadb already installed for hive metastore?

File: dr-elephant/dr-elephant.sh [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+function prepare_mysql(){
+  systemctl restart mysql
+  mysql -u root -proot-password -e " \
+  CREATE DATABASE drelephant;"
+}
+
+
+function run_dr(){
+  # Restart History Server
+  bash $SPARK_HOME/sbin/stop-history-server.sh
```
Comment (karth295) [inline dr-elephant/dr-elephant.sh]: Nit: `systemctl restart spark-historyserver`

File: dr-elephant/test.sh [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+gcloud dataproc clusters create ${cluster_prefix}-ha-${suffix} --subnet default \
+  --zone us-central1-a --initialization-action-timeout 25m\
+  --num-masters 3 --master-machine-type n1-standard-4 --master-boot-disk-size 500 \
+  --num-workers 2 --worker-machine-type n1-standard-4 --worker-boot-disk-size 500 \
+  --image-version ${image} --project ${project_name} \
+  --initialization-actions "${bucket_name}/dr-elephant/dr-elephant-env.sh","${bucket_name}/dr-elephant/dr-elephant.sh" &
+
+# 1.2
+image='1.2'
+suffix='12'
```
Comment (karth295) [inline dr-elephant/test.sh]: Nit: use dashes for all these suffixes to make it clear that it's not the number twelve.

**Comment by adam-stasiak** [comment]
@karth295 Suggestions are applied right now.

**Pull request merged**